pkg/v1/providers/ytt/03_customizations/02_avi/add-avi-labels.yaml

```diff
@@ -0,0 +1,20 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:json", "json")
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:template", "template")
+
+
+#! Label workload cluster if AVI_LABELS is specified
```

Comment:
So we can use AVI_LABELS flag to label cluster during creation? Why we name it with AVI_ prefix?

Reply:
we use the AVI_ prefix to comply with the existing `AVI_LABELS` for mgmt clusters. 

* when `AVI_LABELS` is specified for MC, the install-ako-for-all ADC will have `clusterSelector` set
* when `AVI_LABELS` is specified for WC, the cluster object will be labelled to be selected by the ADC above

Reply:
Got it, thanks